llama-index-networks/pyproject.toml

```diff
@@ -32,7 +32,7 @@ maintainers = [
 name = "llama-index-networks"
 packages = [{include = "llama_index/"}]
 readme = "README.md"
-version = "0.2.0"
+version = "0.2.1a2"
```

Comment:
Do you want this to stay as an alpha?

Reply:
thanks! nope this needs to be upgraded to `0.2.1`